admin/proc-rhdh-deploy-aks-using-operator.adoc

```diff
@@ -85,6 +85,45 @@ kubectl -n <your_namespace> apply -f rhdh-ingress.yaml
 ----
 --
 
+. Create a ConfigMap named `app-config-rhdh` containing the {product-short} configuration using the following template:
++
+--
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: app-config-rhdh
+data:
+  "app-config-rhdh.yaml": |
+    app:
+      title: Red Hat Developer Hub
+      baseUrl: https://<app_address>
+    backend:
+      auth:
+        keys:
+          - secret: "${BACKEND_SECRET}"
+      baseUrl: https://<app_address>
```

Comment:
It would be good to have a unified naming convention for _<app_address>_ vs. _<rhdh_dns_name>_. In EKS configuration, the same entities are listed like below. 

```
data:
  "app-config-rhdh.yaml": |
    app:
      title: Red Hat Developer Hub
      baseUrl: https://<rhdh_dns_name>
    backend:
      auth:
        keys:
          - secret: "${BACKEND_SECRET}"
      baseUrl: https://<rhdh_dns_name>
      cors:
        origin: https://<rhdh_dns_name>
```

Reply:
Agree, and I'd propose to use "*_address" (not "dns") because It can be either DNS or IP